### DISCUSSION TITLE
need help widget - delay after load not work?

### DISCUSSION THREAD

**Discussion by comsyspro** [LiveHelperChat/livehelperchat#1845] [General]
in the widget themes there is a setting in the need help widget `after how many seconds after page load show need help tooltip?`. if i enter here a value e.g. 10 i normally would expect that the need help widget will show up after a 10 seconds delay. but that isn't the case and the need help widget shows up immediatly. i use new embeed widget code. is this a bug?

![image](https://user-images.githubusercontent.com/13809377/207318155-401572e9-2d44-49df-b815-8d42f853e7aa.png)

**Comment by remdex** [comment]
Yes, looks like bug. I'll fix in a hour or so if all goes well :)

**Comment by remdex** [comment]
Try to update and all should be fine now.

**Comment by comsyspro** [comment]
yes it works now, thank you. i also found another bug, and please look into that code https://github.com/LiveHelperChat/livehelperchat/issues/1833
it seems that there are two lines where the translation is not working correct and have to be changed in `$translationInstance->getTranslation('chat/getstatus', 'Our staff are always ready to help')`
